setup.py

```diff
@@ -8,13 +8,9 @@
     author="Canonical webteam",
     author_email="webteam@canonical.com",
     url="https://github.com/canonical-webteam/blog-flask-extension",
-    description=(
-        "Flask extension to add a nice blog to your website"
-    ),
+    description=("Flask extension to add a nice blog to your website"),
     packages=find_packages(),
     long_description=open("README.md").read(),
-    install_requires=[
-        "canonicalwebteam.http==1.0.1",
-        "Flask==1.0.2",
-    ],
+    install_requires=["canonicalwebteam.http==1.0.1", "Flask==1.0.2"],
```

Comment:
Is flask dependency needed?

Reply:
I believe it is, you probably need django to. Have a look at https://github.com/benoitc/gunicorn/blob/master/setup.py.
The extra_requires option in particular might be useful. For gunicorn if you do pip install gunnicorn['gevent'] it will install the dependecies listed under  `extra_required['gevent']`. We could do the same `pip install blog-extension['flask']` or `pip install blog-extension['django']` and make flask and django an extra require in those keys.

Reply:
Ive addded both Django and Flask to the extra-requires